deploy-manage/tools/snapshot-and-restore/manage-snapshot-repositories.md

```diff
@@ -4,4 +4,62 @@
+* [AWS S3](/deploy-manage/tools/snapshot-and-restore/ec-aws-custom-repository.md)
+* [Source-only](/deploy-manage/tools/snapshot-and-restore/source-only-repository.md)
+
+For more details, refer to [Managing snapshot repositories in Elastic Cloud Hosted](/deploy-manage/tools/snapshot-and-restore/elastic-cloud-hosted.md).
+
+### Elastic Cloud Enterprise (ECE)
+
+Snapshot repositories are managed at the platform level in Elastic Cloud Enterprise (ECE) and can be associated with deployments as needed. ECE supports the creation and maintenance of multiple repositories, but each deployment can be linked to only one repository for automatic snapshots.
+
+When a platform-level repository is associated with a deployment, the `found-snapshots` repository is added to the {{es}} cluster, and, similar to Elastic Cloud, a snapshot is taken every 30 minutes by default. The interval can be adjusted on per deployment basis.```
+
+Elastic Cloud Enterprise installations support the following Elasticsearch snapshot repository types:
+
+* [Azure](/deploy-manage/tools/snapshot-and-restore/azure-storage-repository.md)
+* [Google Cloud Storage](/deploy-manage/tools/snapshot-and-restore/google-cloud-storage-gcs-repository.md)
+* [AWS S3](/deploy-manage/tools/snapshot-and-restore/minio-on-premise-repository.md)
+
+:::{note}
+No repository types other than those listed are supported in the Elastic Cloud Enterprise platform, even if they are supported by Elasticsearch. 
+:::
+
+For more details, refer to [Managing snapshot repositories in Elastic Cloud Enterprise](/deploy-manage/tools/snapshot-and-restore/cloud-enterprise.md).
+
+### Self-managed
+
+If you manage your own Elasticsearch cluster, you can use the following built-in snapshot repository types:
+
+* [Azure](/deploy-manage/tools/snapshot-and-restore/azure-repository.md)
+* [Google Cloud Storage](/deploy-manage/tools/snapshot-and-restore/google-cloud-storage-repository.md)
+* [AWS S3](/deploy-manage/tools/snapshot-and-restore/s3-repository.md)
+* [Shared file system](/deploy-manage/tools/snapshot-and-restore/shared-file-system-repository.md)
+* [Read-only URL](/deploy-manage/tools/snapshot-and-restore/read-only-url-repository.md)
+* [Source-only](/deploy-manage/tools/snapshot-and-restore/source-only-repository.md)
+
+Other repository types are available through official plugins:
+
+* [Hadoop Distributed File System (HDFS)](https://www.elastic.co/guide/en/elasticsearch/plugins/current/repository-hdfs.html)
+
+#### Elastic Cloud on Kubernetes (ECK)
+
+ECK is also considered a self-managed deployment method. It supports the same snapshot repositories as self-managed Elasticsearch and has no additional restrictions.
+
+For more details, refer to the Managing snapshot repositories in [Self managed deployments](/deploy-manage/tools/snapshot-and-restore/self-managed.md) and the [Elastic Cloud on Kubernetes](/deploy-manage/tools/snapshot-and-restore/cloud-on-k8s.md) documentation.
```

Comment:
I wouldn't link specifically the self-managed option, better to link only the ECK related doc which has examples and details.

```suggestion
For more information and examples, refer to [create automated snapshots on ECK](/deploy-manage/tools/snapshot-and-restore/cloud-on-k8s.md).
```